buildbot.mariadb.org/util/nginx.conf

```diff
@@ -22,6 +22,8 @@ server {
 	# put a one day session timeout for websockets to stay longer
 	ssl_session_cache   shared:SSL:10m;
 	ssl_session_timeout 1d;
+        # Enable session tickets
+        ssl_session_tickets on;
```

Comment:
I wasn't aware of **session_tickets**, however, brief googling yields:
https://github.com/mozilla/server-side-tls/issues/135

It might be best to stay away from it.